src/RepairMap.vue

```diff
@@ -128,7 +128,7 @@
           </template>
         </div>
         <div v-if="filters.product_categories.length" class="flex flex-wrap mb-2 -mx-1 -my-2 align-middle">
-          <div class="my-2 ml-1 mr-2">Category:</div>
+          <div class="my-2 ml-1 mr-2">{{$t('category_filters')}}</div>
```

Comment:
Spacing uniform trekken met de vertaling erboven. `{{ $t(...) }}` in plaats van `{{$t(...)}}`.